### DISCUSSION TITLE
Simple Newbie Question- SSH

### DISCUSSION THREAD

**Discussion by dobbywocky** [community/community#147111] [New to GitHub] [answered]
I'm doing a couple of different tutorials from the same device. I won't be going from one computer to another. Do I need separate SSH keys for each repository, or can I use the one I already have linked to? My understanding (which may be inaccurate) is that they are related to the device you're using. If I'm mistaken, please feel free to kindly correct me.

**Comment by Logophoman** [comment, marked as answer]
You can create one SSH key for GitHub and use it for all of your projects on the same computer. I do that with my VSCode. There's plenty Tutorials in how to do it and one Key is enough.

**Comment by tzachi81** [comment]
Hello, dobbywocky.

When you create SSH key on a specific device/machine, it may have both private key and public key: private key should remain secure on that device. The public key can be shared and be served across multiple servers or services you want to access in your different tutorials, for example: GitHub, Bitbucket, [some]OpenAPI...

As for your question, if you use the same service (for example"GitHub") managing your repositories, you want use one/single public key you create. It looks something like: "id_[someSuffix].pub"

* Of course you can copy your private SSH key to another machine, but it will create a security issue. That practice is not recommended (even if your doing tutorials)
